### PR TITLE
ogr2ogr, ogrinfo: refresh

### PR DESCRIPTION
**Pull request by JakobMiksch**
This pull request updates the pages `ogr2ogr` and `ogrinfo`

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
`GDAL 3.2.2, released 2021/03/05`

### CODE REVIEW

File: pages/common/ogrinfo.md [hunk 1]
```diff
@@ -15,6 +15,18 @@
...
+
+`ogrinfo -so -al {{path/to/input.gpkg}}`
+
+- Show detailed information of features matching a condition:
+
+`ogrinfo -where '{{myProperty > 42}}' {{path/to/input.gpkg}} {{layer_name}}`
+
+- Update a layer in the data source with SQL:
+
+`ogrinfo {{path/to/input.geojson}} -dialect SQLite -sql {{"UPDATE input SET attribute_name='foo'"}}`
```
Comment (navarroaxel) [inline pages/common/ogrinfo.md:32]:
```suggestion
`ogrinfo {{path/to/input.geojson}} -dialect SQLite -sql "{{UPDATE input SET attribute_name = 'foo'}}"`
```

File: pages/common/ogrinfo.md [hunk 2]
```diff
@@ -15,6 +15,18 @@
...
 
 `ogrinfo -so {{path/to/input.gpkg}} {{layer_name}}`
+
+- Show summary of all layers of the data source:
+
+`ogrinfo -so -al {{path/to/input.gpkg}}`
+
+- Show detailed information of features matching a condition:
+
+`ogrinfo -where '{{myProperty > 42}}' {{path/to/input.gpkg}} {{layer_name}}`
```
Comment (navarroaxel) [inline pages/common/ogrinfo.md:28]: is this a property or an attribute?

Comment (JakobMiksch) [reply pages/common/ogrinfo.md:28]: Good catch. I think both terms might be applicable, but `attribute` might be better. I will update it.

Comment (JakobMiksch) [reply pages/common/ogrinfo.md]: I fixed it in [a009629](https://github.com/tldr-pages/tldr/pull/7904/commits/a009629240b89c276b82cfac12f81d32c2488cb2). 
I resolve the converstation. 
Please reopen if you want.

File: pages/common/ogrinfo.md [hunk 3]
```diff
@@ -15,6 +15,18 @@
...
 
 `ogrinfo -so {{path/to/input.gpkg}} {{layer_name}}`
+
+- Show summary of all layers of the data source:
+
+`ogrinfo -so -al {{path/to/input.gpkg}}`
+
+- Show detailed information of features matching a condition:
+
+`ogrinfo -where '{{my_attribute > 42}}' {{path/to/input.gpkg}} {{layer_name}}`
```
Comment (navarroaxel) [inline pages/common/ogrinfo.md:28]: Use same placeholder
```suggestion
`ogrinfo -where '{{attribute_name > 42}}' {{path/to/input.gpkg}} {{layer_name}}`
```

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**